docs/integrations/sources/jina-ai.md

```diff
@@ -0,0 +1,51 @@
+# Jina AI
+
+## Overview
+
+Jina AI Reader API addresses these issues of scraping by extracting the core content from a URL and converting it into clean, LLM-friendly text, ensuring high-quality input for your agent and RAG systems.
+
+### Available Streams
```

Comment:
Suggesting a quick explanation of both APIs for framing at the top of this readme. Something like this?:

```suggestion

This connector allows access to the [Jina Reader API](https://jina.ai/reader/) using two modes:

- **"Reader" Mode** (`https://r.jina.api`) - Given a set of one or more URLs as input, return the content of those pages as Markdown text. The Reader endpoint extracts the core content from a URL and converting it into clean, LLM-friendly text, ensuring high-quality input for your agent and RAG systems.
- **"Search" Mode** (`https://s.jina.api`) - Similar to the reader endpoint, but accepting a search prompt and returning the text from top 5 search results.

Both of these API endpoints will generate human readable markdown, which can also be efficiently processed by downstream LLM and GenAI applications.

Both modes can be utilized in the same sync, following the configuration instructions below.

### Available Streams
```

Reply:
:white_check_mark: